sources/github/helpers.py

```diff
@@ -22,10 +22,15 @@ def _get_auth_header(access_token: str) -> StrAny:
 #
 # Rest API helpers
 #
-def get_rest_pages(access_token: str, query: str) -> Iterator[List[StrAny]]:
-    url = REST_API_BASE_URL + query
-    while True:
+def _get_rest_pages(access_token: str, query: str) -> Iterator[List[StrAny]]:
```

Comment:
you import this function to another script, so imo prefix "_", which means "private function", doesn't make sense. 
you can keep it like `get_rest_pages`.

same for `get_reactions_data`

Reply:
okay

Reply:
done.